extensions/dicom-sr/src/getSopClassHandlerModule.js

```diff
@@ -26,12 +26,16 @@ const CodeNameCodeSequenceValues = {
   ImageLibraryGroup: '126200',
   TrackingUniqueIdentifier: '112040',
   TrackingIdentifier: '112039',
+  Finding: '121071',
```

Comment:
I wish this were easier to read/parse

Reply:
The point here is I'm making a mapping for this.

We could make this a part of the naturalization process of dcmjs, but someone would have to put in the work of naturalizing all these.